smart-scan/src/main/java/PreviousSeasons.java

```diff
@@ -130,170 +60,57 @@ public static int setPWP(String team, String sku) throws IOException {
-            time = time.substring(0, time.indexOf("T"));
-            //System.out.println(time);
-            try
             {
-                DateFormat formatter;
-                formatter = new SimpleDateFormat("yy-MM-dd");
-                previousDateList.add((Date)formatter.parse(time));
-
+                System.out.println("Error, getOldSKU handler - date formatter");
             }
-            catch (Exception e)
-            {}
 
         }
 
-        //System.out.println(previousDateList);
-        //System.out.println(currentDate);
+        
+        
         Date nearest = (getDateNearest(previousDateList, currentDate));
-        //System.out.println(nearest);
+        
         int indexSKUlookup = previousDateList.indexOf(nearest);
-        String oldSKU = (previousdates.get(indexSKUlookup));
-        URL url2 = new URL("http://api.vexdb.io/v1/get_rankings?team="+team + "&sku="+oldSKU);
-        //System.out.println(url2);
+        return(previousdates.get(indexSKUlookup));
+    }
+    public static double setPCCWM(String team, String sku, boolean twoyears) throws IOException {
+        String oldSKU = getOldSKU(team, sku, twoyears);
+        JsonParser jsonParser = new JsonParser();
+        URL url2 = new URL("https://api.vexdb.io/v1/get_rankings?team="+team + "&sku="+oldSKU);
+        
         InputStreamReader reader2 = new InputStreamReader(url2.openStream());
 
         try {
             JsonArray stuff2 = (JsonArray) jsonParser.parse(reader2).getAsJsonObject().get("result"); //idk what the hell this is
-            //System.out.println(stuff2);
-            return(stuff2.get(0).getAsJsonObject().get("ap").getAsInt());
+            
+            return(stuff2.get(0).getAsJsonObject().get("ccwm").getAsDouble());
 
         }
         catch (Exception e) {
-            return(0);
+            return(0.0);
         }
     }
-    public static int setPSP(String team, String sku) throws IOException {
-        ArrayList<String> previousdates = new ArrayList<>();
-        ArrayList<Date> previousDateList = new ArrayList<>();
-        Event eventThing = new Event(sku);
-        eventThing.setSeason();
-        Date currentDate = null;
-        String currentDateString = eventThing.getDatetime();
-        currentDateString = currentDateString.replace("%3A", "");
-        currentDateString = currentDateString.substring(0, currentDateString.indexOf("T"));
-
-        try
-        {
-            DateFormat formatter;
-            formatter = new SimpleDateFormat("yy-MM-dd");
-            currentDate = ((Date)formatter.parse(currentDateString));
-
-        }
-        catch (Exception e)
-        {}
-        URL url = new URL("http://api.vexdb.io/v1/get_events?season=" + eventThing.getPreviousSeason(false) + "&team=" + team);
-        //System.out.println(url);
-        InputStreamReader reader = new InputStreamReader(url.openStream());
+    public static int setPWASP(String team, String sku, String option) throws IOException {
```

Comment:
The lack of documentation around your methods or even a single useful comment in this entire codebase is horrific and you're gonna be very mad at yourself if you come back to this later, and you're going to be embarrassed of yourself if you share this code with a recruiter or colleague.